testsuite/features/step_definitions/command_steps.rb

```diff
@@ -189,7 +193,8 @@
 
 When(/^I query latest Salt changes on ubuntu system "(.*?)"$/) do |host|
   node = get_target(host)
-  result, return_code = node.run("zcat /usr/share/doc/salt-minion/changelog.Debian.gz")
+  salt = $product == 'Uyuni' ? "venv-salt-minion" : "salt-minion"
+  result, return_code = node.run("zcat /usr/share/doc/#{salt}/changelog.Debian.gz")
```

Comment:
In of Debian, the changelog file for `venv-salt-minion` package is slightly different: named `changelog.gz`

```suggestion
  changelog_file = $product == 'Uyuni' ? "changelog.gz" : "changelog.Debian.gz"
  result, return_code = node.run("zcat /usr/share/doc/#{salt}/#{changelog_file}")
```